bigbluebutton-html5/imports/api/meetings/server/handlers/recordingStatusChange.js

```diff
@@ -0,0 +1,29 @@
+import { check } from 'meteor/check';
+import Meetings from '/imports/api/meetings';
+import Logger from '/imports/startup/server/logger';
+
+export default function handleRecordingStatusChange({ body }, meetingId) {
+  const { recording } = body;
+  check(recording, Boolean);
+
+  const selector = {
+    meetingId,
+  };
+
+  const modifier = {
+    $set: { 'recordProp.record': recording },
+  };
+
+  const cb = (err, numChanged) => {
+    if (err) {
+      Logger.error(`Changing record status: ${err}`);
+      return;
+    }
+
+    if (numChanged) {
+      Logger.info(`Changed meeting record status id=${meetingId} recording=${recording}`);
+    }
```

Comment:
You must have a linter error here, saying that you should return something from the arrow function.

Reply:
Please provide the linter error. There's no error on this file.

Reply:
I wonder if updating the package removed it, I just reinstalled node_modules
Don't see this error anymore.